src/main/java/com/jcabi/github/Commits.java

```diff
@@ -70,4 +70,12 @@ Commit create(
      */
     @NotNull(message = "commit is never NULL")
     Commit get(@NotNull(message = "sha can't be null") String sha);
+
+    /**
+     * Return a Statuses object for a given ref (sha, branch name, etc).
+     * @param ref The ref of the Commit
+     * @return Status
```

Comment:
```@since``` is missing. It's important, as we add new method to the API
0.23 is the right one